testnets/testnets.md

```diff
@@ -146,13 +146,18 @@ git checkout -b add-gen-acc-<validator-name>
 cd <testnet-name>
 
 coral keys add validator
-coral init dummy
+corald init dummy
 corald add-genesis-account $(coral keys show -a validator) 100000000ushell,100000000ureef
 cat $HOME/.corald/config/genesis.json | jq '.app_state.auth.accounts [0]' > /tmp/new_acc.json
-NEW_GEN=$(jq '.app_state.auth.accounts += [input]' ./config/genesis.json /tmp/new_acc.json) && echo "$NEW_GEN" > ./config/genesis.json
+NEW_GEN=$(jq '.app_state.auth.accounts += [input]' ./config/genesis.json /tmp/new_acc.json) && echo "$NEW_GEN" > ./config/genesis_new.json
```

Comment:
Thanks for this.
However, the who jq stuff has been deprecated by using the `--home` flag.

We can merge this, but there is another pending cleanup here. I will let @orkunkl finish it: